services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_observer.py

```diff
@@ -1,3 +1,5 @@
+# pylint: disable=relative-beyond-top-level
```

Comment:
are you sure that the pylint configuration is set correctly?

Reply:
let's talk on the above discussion